test/pageobjects/loginPage.js

```diff
@@ -0,0 +1,46 @@
+const BasePage = require(`./basePage.js`);
+const Button = require(`../pageobjects/commonWebComponents/components/button.js`);
+const Input = require(`../pageobjects/commonWebComponents/components/input.js`);
+const Notification = require(`../pageobjects/commonWebComponents/components/notification.js`);
+
+class LoginPage extends BasePage {
+
+    get emailInput() {return new Input('[ng-reflect-name="email"]')}
+    get passwordInput() {return new Input('[ng-reflect-name="password"]')}
+    get submitButton() {return new Button('[class="btn"]')}
+    get notification() {return new Notification('[class="alert-info"]')}
```

Comment:
Move all selector values from the other utils page classes in the selectors.js class when you create it.